test/run_tests.sh

```diff
@@ -188,7 +188,8 @@ function run_xla_op_tests {
   run_test "$CDIR/pjrt/test_mesh_service.py"
   run_test "$CDIR/spmd/test_xla_sharding.py"
   run_test "$CDIR/spmd/test_xla_virtual_device.py"
-  run_test "$CDIR/spmd/test_dynamo_spmd.py"
+  # TODO(yeounoh) SPMD output sharding is blocking Dynamo integration.
```

Comment:
@JackCaoG we need to revisit SPMD+Dynamo integration after output sharding. Let's take it outside the current scope, as it requires some more work & design.